include/mlir/Dialect/Linalg/Utils/Utils.h

```diff
@@ -23,6 +23,10 @@
 #include "mlir/Dialect/StandardOps/Ops.h"
 #include "mlir/EDSC/Helpers.h"
 
+#include "llvm/ADT/SetVector.h"
+
+using llvm::SetVector;
```

Comment:
Don't do this, don't add using directives in a header. Use it in a fully resolved context, or add this to LLVM.h

Reply:
Oooops! Done